src/plugins/dashboard/public/dashboard_actions/copy_to_dashboard_modal.tsx

```diff
@@ -88,90 +92,70 @@ export function CopyToDashboardModal({
-                </EuiText>
-                <EuiSpacer />
-                <EuiFormRow hasChildLabel={false}>
-                  <EuiPanel
-                    color="subdued"
-                    hasShadow={false}
-                    data-test-subj="add-to-dashboard-options"
-                  >
-                    <div>
-                      {canEditExisting && (
-                        <>
-                          <EuiRadio
-                            checked={dashboardOption === 'existing'}
-                            data-test-subj="add-to-existing-dashboard-option"
-                            id="existing-dashboard-option"
-                            name="dashboard-option"
-                            label={dashboardCopyToDashboardActionStrings.getExistingDashboardOption()}
-                            onChange={() => setDashboardOption('existing')}
-                          />
-                          <div className="savAddDashboard__searchDashboards">
-                            <DashboardPicker
-                              isDisabled={dashboardOption !== 'existing'}
-                              idsToOmit={dashboardId ? [dashboardId] : undefined}
-                              onChange={(dashboard) => setSelectedDashboard(dashboard)}
-                            />
-                          </div>
-                          <EuiSpacer size="s" />
-                        </>
-                      )}
-                      {canCreateNew && (
-                        <>
-                          <EuiRadio
-                            checked={dashboardOption === 'new'}
-                            data-test-subj="add-to-new-dashboard-option"
-                            id="new-dashboard-option"
-                            name="dashboard-option"
-                            disabled={!dashboardId}
-                            label={dashboardCopyToDashboardActionStrings.getNewDashboardOption()}
-                            onChange={() => setDashboardOption('new')}
-                          />
-                          <EuiSpacer size="s" />
-                        </>
-                      )}
-                    </div>
-                  </EuiPanel>
-                </EuiFormRow>
-              </>
-            </EuiModalBody>
+      <EuiModalBody>
+        <>
+          <EuiFormRow hasChildLabel={false}>
+            <div data-test-subj="add-to-dashboard-options">
+              {canEditExisting && (
+                <>
+                  <EuiRadio
+                    checked={dashboardOption === 'existing'}
+                    data-test-subj="add-to-existing-dashboard-option"
+                    id="existing-dashboard-option"
+                    name="dashboard-option"
+                    label={dashboardCopyToDashboardActionStrings.getExistingDashboardOption()}
+                    onChange={() => setDashboardOption('existing')}
```

Comment:
super nit-picky thing but not a blocker

It would be nifty if when "New dashboard" is selected and I select a dashboard in the popover that the radio button would automatically switch to "Existing dashboard"


https://github.com/elastic/kibana/assets/1638483/21b7065a-86c0-4694-9033-fdf029822eff

Reply:
Good idea! Addressed this in 0584f4e1f43